### ISSUE TITLE
[Documentation] How to update Freshen, on a running Gobolinux system?

### ISSUE THREAD

**Issue by rubyFeedback** [gobolinux/Freshen#4]
Could this question be answered in the main README?

"How to update Freshen, on a running Gobolinux system?"

I assume Freshen has some option such as -u or something
like that. (It may actually be useful to have the documentation
also available on github, if only as an additional resource.)

**Comment by fyrak1s** [comment]
Try go in /Programs/Freshen/Current and git pull.

Not sure it's a git archive right out of my head, but it could be.

**Comment by fyrak1s** [comment]
Actually, the correct way is to install it using `Compile Freshen git`. But that's broken currently, please patiently wait for the fix.

**Comment by fyrak1s** [comment]
@rubyFeedback It should be fixed now (https://github.com/gobolinux/Recipes/pull/223). Please try whether `Compile Freshen git` works for you. Your feedback is valuable.

But do also note that Freshen is very experimental, and not yet officially supported. I suggest not running `Freshen -U`, apparently it can break systems :) But in a safe environment like a VM it might be fine.

TL;DR: Have fun but keep it safe:)

**Comment by fyrak1s** [comment]
No feedback so assuming it is fixed. Closing.
